components/composer/src/Composer.svelte

```diff
@@ -677,14 +717,21 @@
         <!-- Search -->
         <div class="contacts-wrapper">
           {#if _this.show_from}
-            <nylas-contacts-search
-              data-cy="from-field"
-              placeholder="From:"
-              single={true}
-              change={handleContactsChange("from")}
-              contacts={from}
-              value={$message.from}
-            />
+            <div class="contacts-container">
+              <div class="contact-from">
+                <div class="contacts-placeholder">From:</div>
+                <div class="contacts-results-inner">
+                  <div class="contact-item" data-cy="from-field">
+                    <span class="contact-item__name">
+                      {#if $message.from[0].name}
+                        <strong>{$message.from[0].name}</strong>
+                        {`<${$message.from[0].email}>`}
+                      {:else}{$message.from[0].email}{/if}
+                    </span>
+                  </div>
+                </div>
+              </div>
```

Comment:
@hazik @pooja169usp so we're never supporting changing of the from field?

Reply:
@AaronDDM we could add a prop that would enable/disable changing form field. 
But I wonder, when would someone want to change the from field? It is tied to the access token.